api/api.go

```diff
@@ -179,7 +179,11 @@ func (w Wrapper) GetServicesForCustomer(ctx echo.Context, customerID string) err
 	if err != nil {
 		return err
 	}
-	return ctx.JSON(http.StatusOK, services)
+	response := make([]did.Service, len(services))
```

Comment:
why the copy?

Reply:
I need an initialized array so JSON serialization gives an `[]` instead of `null` when there are no services for this customer.